src/dsp_tools/commands/xmlupload/models/ontology_lookup_models.py

```diff
@@ -18,39 +18,39 @@ class ProjectOntosInformation:
     onto_lookup: dict[str, OntoInfo]
 
 
-def extract_classes_properties_from_onto(onto_graph: list[dict[str, Any]]) -> OntoInfo:
+def extract_classes_properties_from_onto(onto_json: list[dict[str, Any]]) -> OntoInfo:
```

Comment:
Grammatically, it is not correct to say: "Extract classes properties from onto". It is necessary to have an "and" between the 2 things that you extract from the onto.